EssentialFeed/EssentialFeedTests/Image Comments/API/ImageCommentsMapperTests.swift

```diff
@@ -0,0 +1,75 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentsMapperTests: XCTestCase {
+	func test_map_throwsErrorOnNon200HTTPResponse() throws {
```

Comment:
The name should be `test_map_throwsErrorOnNon2xxHTTPResponse` instead because 201..<300 are considered to be valid codes, not only 200
```suggestion
	func test_map_throwsErrorOnNon2xxHTTPResponse() throws {
```

Reply:
Renamed.